packages/terra-graphs/src/terra-dev-site/doc/Graphs/Graphs.1.doc.mdx

```diff
@@ -0,0 +1,36 @@
+import { Badge } from '@cerner/terra-graphs/package.json?dev-site-package';
+
+import DefaultGraphs from '../example/DefaultGraphs?dev-site-example';
+
+import PropsTable from '@cerner/terra-graphs/src/Graphs?dev-site-props-table';
+
+<Badge />
+
+# Terra Graphs
+
+{insert description}
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-graphs`
```

Comment:
Do we need to add the Peer Dependency section? 
https://engineering.cerner.com/terra-ui/components/terra-alert/alert/alert#peer-dependencies

Reply:
Lets just phase that peer dependency section out

Reply:
I am 100% good with that. That info is available in the package.json.